packages/@atjson/offset-annotations/src/annotations/accordion.ts

```diff
@@ -0,0 +1,28 @@
+import { BlockAnnotation } from "@atjson/document";
+
+export class Accordion extends BlockAnnotation<{
```

Comment:
Could you add a comment that links to the WAI-ARIA spec that specifies the `header` / `panel` properties?

Reply:
added pls review